gladier_xpcs/deployments.py

```diff
@@ -138,11 +138,25 @@ class HannahDeployment(BaseDeployment):
 
     flow_input = {
         'input': {
-            'staging_dir': '/projects/APSDataAnalysis/hannah/gladier_testing/',
+            'staging_dir': '/eagle/projects/APSDataAnalysis/XPCS/hparraga/gladier_testing/',
             'corr_loc': '/eagle/APSDataAnalysis/XPCS/xpcs-eigen/build/corr',
         }
     }
 
+class HannahPolaris(HannahTheta):
+    funcx_endpoints = {
+        'funcx_endpoint_non_compute': 'e3e1aef6-0a6f-4ef1-b9c6-a14b0efb1dfa',
+        'funcx_endpoint_compute': '0bbf9fe3-0cae-43bd-9307-625c9b07e3b6',
+    }
+
+    flow_input = {
+        'input': {
+            'staging_dir': '/eagle/projects/APSDataAnalysis/XPCS/hparraga/gladier_testing/',
+            'corr_loc': '/eagle/APSDataAnalysis/XPCS/xpcs-eigen/build/corr',
+            'corr_gpu_loc' : '/eagle/projects/APSDataAnalysis/XPCS/mchu/xpcs_boost/gpu_corr.py',
```

Comment:
Same here (and for all other deployments)
@NickolausDS maybe we should try that on the CPU flow today